assets/src/stories-editor/components/context-menu/index.js

```diff
@@ -201,6 +200,8 @@ const ContextMenu = ( props ) => {
 		left: clientX,
 	};
 
+	const POPOVER_POSITION = ( isRTL ) ? 'bottom right' : 'bottom left';
```

Comment:
It really shouldn't be `SCREAMING` if it's not a constant. And it's just a local variable.

```suggestion
	const popoverPosition = ( isRTL ) ? 'bottom right' : 'bottom left';
```

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3633/commits/4e5eadff03f9486e5ad8c91f54c679dbe7687ba2